sdk/rust-examples/sort-numbers/Cargo.lock

```diff
@@ -0,0 +1,16 @@
+# This file is automatically @generated by Cargo.
```

Comment:
We do not need to commit Cargo.lock; there is one center `cargo.lock` under workspace directory